components/server/src/server.rs

```diff
@@ -1475,14 +1517,45 @@ where
                     .get_engine_size()
                     .expect("get raft engine size");
 
+                let mut raft_disk_status = disk::DiskUsage::Normal;
+                if separated_raft_mount_path {
```

Comment:
minor: I think it might be better to use reserve_raft_space != 0 instead of "separated_raft_mount_path" so that when it's configured to 0, the following block can be skipped completely.

Reply:
Thanks for your suggestion - will check separated_raft_mount_path and reserve_raft_space here.